how-to.html

```diff
@@ -0,0 +1,80 @@
+---
+layout: two-col
+---
+
+<div class="margin-trailer">
+  <h2>What to Share</h2>
+  <p>
+    Stories submitted to The Second Page should be largely true, and involve your own experiences.
+    We welcome stories of all sizes (a few minutes up to half an hour). Recordings are preferred in WAV or AIF format,
+    though MP3s are okay. Recordings should be read aloud by you or a friend.
+  </p>
+  <p>
+    If you would like to contribute, but cannot read your story or prefer to remain anonymous, let us know.
+    If aired, we can likely include your work under another name, and record it ourselves.
```

Comment:
Would simplify:

~If aired,~ **W**e can ~likely~ include your work under another name, and record it ourselves.